pymc/distributions/shape_utils.py

```diff
@@ -618,4 +618,4 @@ def find_size(
 
 def rv_size_is_none(size: Variable) -> bool:
     """Check wether an rv size is None (ie., at.Constant([]))"""
-    return size.type.shape == (0,)
+    return size.type.shape == (0,) # type: ignore [attr-defined]
```

Comment:
Ideally this should be fixed upstream, but I didn't want to put `shape_utils.py` onto the list of non-passing files because for `shape_utils.py` it's quite valuable to enforce mypy checks.